includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -1114,7 +1114,9 @@ public function process_payment_for_order( $cart, $payment_information, $schedul
 				$request->set_off_session( $payment_information->is_merchant_initiated() );
 				$request->set_payment_methods( $payment_methods );
 				$request->set_cvc_confirmation( $payment_information->get_cvc_confirmation() );
-				$request->set_shipping( $this->get_shipping_data_from_order( $order ) );
+				if ( 'afterpay_clearpay' === $payment_information->get_payment_method() ) {
```

Comment:
Reminder: constants from 'Payment_Method'

Reply:
Thanks! changed it :) 
dd09c43